runners.go

```diff
@@ -414,3 +414,39 @@ func (s *RunnersService) VerifyRegisteredRunner(opt *VerifyRegisteredRunnerOptio
 
 	return s.client.Do(req, nil)
 }
+
+// ListGroupsRunnersOptions represents the available ListGroupsRunners() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/runners.html#list-groups-runners
+type ListGroupsRunnersOptions struct {
+	ListOptions
+	Type    *string  `url:"type,omitempty" json:"type,omitempty"`
+	Status  *string  `url:"status,omitempty" json:"status,omitempty"`
+	TagList []string `url:"tag_list,comma,omitempty" json:"tag_list,omitempty"`
+}
+
+// ListGroupsRunners List all runners (specific and shared) available in the group as well it’s ancestor groups.
+// Shared runners are listed if at least one shared runner is defined.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/runners.html#list-groups-runners
+func (s *RunnersService) ListGroupsRunners(gid interface{}, opt *ListGroupsRunnersOptions, options ...RequestOptionFunc) ([]*Runner, *Response, error) {
```

Comment:
The methods in this file are ordered using the order in which they are documented. So following that logic, this new code should be placed below the `DisableProjectRunner` method.

Reply:
Thanks!